src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -51,7 +54,7 @@
   {:render
    (fn [{:keys [props]}]
      (let [{:keys [nav-context data]} props
-           {:keys [href status protected? disabled?]} data
+           {:keys [href status restricted? disabled? protected?]} data
```

Comment:
The property called `protected?` has been renamed to `restricted?` because:

1. It has nothing to do with the workspace.isProtected flag returned from orchestration
2. It is used to conditionally show the rotated "RESTRICTED" indicator in the workspace list

Then, I re-added `protected?` which actually does come from the workspace.isProtected flag.